CONTRIBUTING.asciidoc

```diff
@@ -276,8 +276,8 @@ Hints
 Python and Qt objects
 ~~~~~~~~~~~~~~~~~~~~~
 
-For many tasks, there are solutions in both Qt and the Python standard library
-available.
+For many tasks, there are solutions available in both Qt and the Python
+standard library available.
```

Comment:
Whoops >.< This is what I get for watching netflix while proofreading. Lemme fix that real quick.